packages/node-sdk/README.md

```diff
@@ -47,46 +47,84 @@ client.initialize().then({
 })
 ```
 
-Once the client is initialized, you can obtain features along with the `isEnabled` status to indicate whether the feature is targeted for this user/company:
+Once the client is initialized, you can obtain features along with the `isEnabled`
+status to indicate whether the feature is targeted for this user/company:
 
-```ts
+```typescript
 // configure the client
 const boundClient = client.bindClient({
   user: { id: "john_doe", name: "John Doe" },
   company: { id: "acme_inc", name: "Acme, Inc." },
 });
 
-// get the huddle feature using company, user and custom context to evaluate the targeting.
+// get the huddle feature using company, user and custom context to
+// evaluate the targeting.
 const { isEnabled, track } = boundClient.getFeature("huddle");
 
 if (isEnabled) {
   // this is your feature gated code ...
   // send an event when the feature is used:
   track();
 
-  // CAUTION: if you plan to use the event for automated feedback surveys call `flush` immediately
-  // after `track`. It can optionally be awaited to guarantee the sent happened.
+  // CAUTION: if you plan to use the event for automated feedback surveys
+  // call `flush` immediately after `track`. It can optionally be awaited
+  // to guarantee the sent happened.
   boundClient.flush();
 }
 ```
 
 You can also use the `getFeatures` method which returns a map of all features:
 
-```ts
-// get the current features (uses company, user and custom context to evaluate the features).
+```typescript
+// get the current features (uses company, user and custom context to
+// evaluate the features).
 const features = boundClient.getFeatures();
 const bothEnabled =
   features.huddle?.isEnabled && features.voiceHuddle?.isEnabled;
 ```
 
-When using `getFeatures` be careful not to assume that a feature exists, this could be a dangerous pattern:
+When using `getFeatures` be careful not to assume that a feature exists, this could
+be a dangerous pattern:
 
 ```ts
 // warning: if the `huddle` feature does not exist because it wasn't created in Bucket
-// or because the client was unable to reach our servers for some reason, this will cause an exception:
+// or because the client was unable to reach our servers for some reason, this will
+// cause an exception:
 const { isEnabled } = boundClient.getFeatures()["huddle"];
 ```
 
+## Opting out of tracking
+
+There are use cases in which one does not want to be sending `user`, `company` and
```

Comment:
Rest of the README is using `you` instead of `one`.

Reply:
Switched to `you`